### PR TITLE
Update XC.9 Emergency Operations

### PR DESCRIPTION
**Pull request by johnjackoleary**

### CODE REVIEW

File: Lesson Plans/Private/XC.9 Emergency Operations.md [hunk 1]
```diff
@@ -23,91 +23,98 @@ Develop knowledge of all abnormal and emergency procedures for the training airc
...
 	1. When the outcome of the flight is in doubt
 	2. When you need higher priority handling from ATC
 
-##### Something is Wrong!
-DISCLAIMER: This bulleted list is intended to help you think through an emergency in a logical way, debug the problem, and understand at a high level how you'll handle the situation.  "**(POH)**" means most POHs have a specific procedure for this emergency.  When that procedure exists, you should follow it, and in many cases memorize it to be prepared.  The steps after "**(POH)**" summarize the high-level purpose of the POH procedure.
+#### Something is Wrong!
+> [!summary]
+> Though rare, emergencies can occur in any phase of flight and can have a variety of causes.
+> 
+> This lesson is intended to help you handle a variety of issues and emergencies. Some emergencies require immediate actions. Other are problems you may be able to debug and resolve using your understanding of aircraft systems. If you can't resolve an issue, it's important to know the impact when deciding to continue or terminate your flight.
```
Comment (johnjackoleary) [inline Lesson Plans/Private/XC.9 Emergency Operations.md:36]: Reorder and reword

**Review by johnjackoleary — COMMENTED** [review]
bit of cleanup

File: Lesson Plans/Private/XC.9 Emergency Operations.md (hunk 1 above)
Comment (johnjackoleary) [reply Lesson Plans/Private/XC.9 Emergency Operations.md]: dne

**Comment by petercmonaco** [comment]
I reviewed and these changes look good.  Thanks, Jack.

**Pull request merged**